assets/stylesheets/sass/components/_fonts.scss

```diff
@@ -0,0 +1,48 @@
+@font-face {
+    font-family: 'Merriweather';
+    src: url('../../../fonts/merriweather-bold-webfont.eot');
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Prefer double-quoted strings